doc/guides/device_mgmt/ota.rst

```diff
@@ -0,0 +1,65 @@
+.. _ota:
+
+Over-the-Air Update
+###################
+
+Overview
+********
+
+Over-the-Air (OTA) Update is a method for delivering firmware updates to remote
+devices using a network connection. Although the name implies a wireless
+connection, devices receiving updates over a wired connection (such as Ethernet)
+are still commonly referred to by OTA. This approach requires server
```

Comment:
This doesn't parse. The devices aren't referred to by "OTA", the updates are. How about this instead:

```suggestion
connection, updates received over a wired connection (such as Ethernet)
are still commonly referred to as OTA updates. This approach requires server
```

Reply:
Good feedback, thanks! I'll incorporate this change in an update to the commit.